src/apps/AppFactory.h

```diff
@@ -26,6 +26,7 @@ class AppFactory {
 	enum class Platform {
 		CPU,
 		GPU,
+		CPUModes,
```

Comment:
This Platform enum does not exist anymore. Again, first resolve your confilcts with the master